lib/HTTP/Server/Tiny.pm6

```diff
@@ -139,6 +139,10 @@ my class HTTP::Server::Tiny::Handler {
             %!env<SERVER_NAME> = $.host;
             %!env<SERVER_PORT> = $.port;
             %!env<SCRIPT_NAME> = '';
+            %!env<p6w.version> = Version.new('v0.7.Draft');
+            %!env<p6w.multithread> = False;
```

Comment:
Though I'm not sure, this value should be `True` since HTTP::Server::Tiny handles requests by `react` and `IO::Socket::Async`?